src/bin/common/solving_loop.ml

```diff
@@ -70,7 +74,7 @@ let main () =
           Options.Time.set_timeout ~is_gui:false (Options.get_timelimit ());
         end;
       SAT.reset_refs ();
-      let _ =
+      let sat_env, _, _ =
```

Comment:
Thinking about it over lunch: `sat_env` is a really confusing name. `partial_model` would better describe what this actually is.